### PR TITLE
fix(isAuthenticated): always check access_token existence

### LINKED ISSUE #269: authenticated and isAuthenticated() always true
authenticated and isAuthenticated() always returning true, altough there is token in session storage. 
 i'm using getExpirationDateFromResponse like

`
getExpirationDateFromResponse: serverResponse => {

```
    let expires: number = 0;
    if (serverResponse != null && serverResponse.expires_in != null) {
        expires = (new Date().getTime() + serverResponse.expires_in * 1000);
    }
    else {
        expires = (new Date().getTime() + 30 * 60 * 1000);             
    }         
    console.log("token expiring in: " + expires);
    return expires;
},  `
```

### PR DESCRIPTION
**Pull request by doktordirk**
fixes https://github.com/SpoonX/aurelia-authentication/issues/269

### CODE REVIEW

**Review by lpj145 — APPROVED** [review]

File: src/authService.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ export class AuthService {
       if (this.config.autoUpdateToken
         && this.authentication.getAccessToken()
         && this.authentication.getRefreshToken()) {
-        this.updateToken();
+        this.updateToken().catch(error => LogManager.getLogger('authentication').warn(error.message));
```
Comment (RWOverdijk) [inline src/authService.js:149]: Please also return error after logging it.

File: src/authService.js [hunk 2]
```diff
@@ -281,7 +281,7 @@ export class AuthService {
       && this.authentication.getAccessToken()
       && this.authentication.getRefreshToken()
     ) {
-      this.updateToken();
+      this.updateToken().catch(error => LogManager.getLogger('authentication').warn(error.message));
```
Comment (RWOverdijk) [inline src/authService.js:288]: Please also return error after logging it.

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

File: src/authService.js (hunk 2 above)
Comment (doktordirk) [reply src/authService.js:288]: can.t that.s an empty promise hehe

File: src/authService.js (hunk 1 above)
Comment (doktordirk) [reply src/authService.js:149]: can.t that.s an empty promise hehe

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**